cluster-soak-tests/src/main/resources/logback.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+
+    <appender name="CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
+        <target>System.out</target>
+        <encoder>
+            <pattern>%date{MM/dd HH:mm:ss} %-5level[%thread] %logger{1} - %m%n%xException</pattern>
```

Comment:
might be useful to include `%X{sourceThread}`, `%X{akkaSource}`, `%X{sourceActorSystem}`
https://doc.akka.io/docs/akka/current/logging.html#logging-thread-akka-source-and-actor-system-in-mdc

What is that `%xException` at the end?

Reply:
:+1: added source/thread. system will be the same for all of them. Not sure where that Exception came from.